syslog-ng/SOURCES/syslog-ng.init

```diff
@@ -12,15 +12,18 @@ source /etc/init.d/kaosv
 ###############################################################################
 
 kv[prog_name]="syslog-ng"
+kv[log]="/var/log/syslog-ng.log"
 
 kv.readSysconfig
 
 binary="${BINARY:-/usr/sbin/syslog-ng}"
 conf_file="${CONF_FILE:-/etc/syslog-ng/syslog-ng.conf}"
 options="$OPTIONS"
+persist_file="${PERSIST_FILE:-/var/lib/syslog-ng/syslog-ng.persist}"
 
 kv[search_pattern]="$binary"
-kv[log]="/var/log/syslog-ng.log"
+kv[user]="${USER:-root}"
```

Comment:
I think you should add method `prepare` ([example](https://github.com/essentialkaos/kaos-repo/blob/master/redis/SOURCES/redis.init#L39) from redis package) and check `user`, `binary` , `conf_file` and `persist_file` variables before script starts any command handlers.